Formula/tbb.rb

```diff
@@ -81,6 +84,10 @@ def install
     assert_path_exists lib/"libtbb.a"
     assert_path_exists lib/"libtbbmalloc.a"
 
+    # tbbbind not support for macOS,
+    # see https://github.com/oneapi-src/oneTBB/blob/690aaf497a78a75ff72cddb084579427ab0a8ffc/CMakeLists.txt#L226-L228
+    assert_path_exists lib/"libtbbbind_2_5.so" if OS.linux?
```

Comment:
This doesn't test if it works, it just tests if it exists

Reply:
added more comprehensive test.